iothub/device/src/Microsoft.Azure.Devices.Client.csproj

```diff
@@ -52,7 +52,7 @@
   <ItemGroup>
     <PackageReference Include="DotNetty.Codecs.Mqtt" Version="0.6.0" />
     <PackageReference Include="DotNetty.Handlers" Version="0.6.0" />
-    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.4.4" />
+    <PackageReference Include="Microsoft.Azure.Amqp" Version="2.4.7" />
```

Comment:
Is this update still compatible with all the frameworks we target?

What fixes come with it?

Reply:
Yes. I ran through the pipelines and everything seems to be in place. 
Here are the fixes we will be bringing in: 
- Implemented timeout when creating a tcp transport. On some platforms the tcp connect can take up to 90 seconds to error out when the remote peer is down.
- Fails to open an AMQPS connection on .NET 5.0.
- Add netstandard2.0 target.
- Dispose timer objects explicitly instead of relying on finalizer.
Notes: for netstandard2.0 target, serialization of a field uses reflection instead of IL emit. If you have fields marked as [AmqpMember], please convert them to property members. It only affect code that uses AmqpContractSerializer.

Reply:
https://github.com/Azure/azure-amqp/releases